Sources/Misc/Signing.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  Signing.swift
+//
+//  Created by Nacho Soto on 1/13/23.
+
+import Foundation
+import Security
+
+/// Utilities for handling certificates and keys.
+enum Signing {
+
+    /// A type object representing an X.509 certificate.
+    typealias Certificate = SecCertificate
+    /// An object that represents a cryptographic key.
+    typealias PublicKey = SecKey
+
+    static let keyAlgorithm: SecKeyAlgorithm = .ecdsaSignatureMessageX962SHA512
+    private static let saltSize = 16
+
+    /// Parses the binary `key` and returns a `PublicKey`
+    /// - Throws: ``ErrorCode/configurationError`` if the certificate couldn't be loaded.
+    @available(iOS 12.0, macCatalyst 13.0, tvOS 12.0, macOS 10.14, watchOS 6.2, *)
```

Comment:
/wtd add example usage of this method to the docstring

Reply:
lol that's terrible

Reply:
for any curious readers, that's not an actual key from anything, the AI made it up 😂